contributing_to_docs/contributing.adoc

```diff
@@ -204,13 +211,16 @@ topic or topic group will appear in all product documentation versions.
 _openshift-origin,openshift-enterprise,openshift-online,openshift-dedicated,openshift-aro,openshift-webscale_.
 * The *all* value overrides other values, so _openshift-online,all_ is processed
 as *all*.
+* Do not use _openshift-dedicated_ or _openshift-rosa_ in the main `&#95;topic&#95;map.yml` file. Use the distribution specific `&#95;topic&#95;map.yml`.
```

Comment:
You could change "Use the distribution specific `_topic_map.yml`" to "Use the distribution specific topic map file" or something similar, given that they have different names.